### ISSUE TITLE
[Question] is there a meta discussion place on the NixOS wiki?

### ISSUE THREAD

**Issue by Narice** [nix-community/wiki#37]
I would like to discuss and read meta discussions on the NixOS wiki, is it something that could be directly available on the wiki?
That would be fantastic!
I have seen that there is this git lhub and only an IRC for discussion but IRC doesn't provide logs so it might be useful to create a discord server, a matrix channel, or maybe a rocket chat? I don't really know what would be best.

**Comment by Mic92** [comment]
I guess it could be bridged with other protocols?

**Comment by Mic92** [comment]
There are also talk pages for articles.

**Comment by Narice** [comment]
Yes, all of the above mentionned can be bridged with irc.

Talk pages are nice for articles but are not enough in my opinion for talking about the wiki in general.

What I was talking about was meta discussions about the wiki so discussions not directly content related.
An example of one would be talking about the rules of the wiki, whaf should be the does and don'ts what is the direction we want to take with the wiki, what pages should be created in what priorityz how to relay that information to wiki viaitors so that they can help if they want to etc x)

Basically, everything that I cannot put inside an issue because they are not issues of deployement or developement, but issues of community and wiki's purpose.

This question issue would also fall under this category for example.

**Comment by Narice** [comment]
We could also use a discourse, that seems to work nicely for the Nix/OS project

**Comment by Mic92** [comment]
Looks like we have enough platforms to choose from.

**Comment by Narice** [comment]
I don't really understand why this issue was closed with your last comment @Mic92, could you elaborate on what you meant?

**Comment by Mic92** [comment]
I don't see any actionable item here. We have the irc, discourse and wiki article discussion pages. IRC can be also accessed via matrix. I think those are enough channels.

**Comment by Narice** [comment]
There is a Discourse for the wiki and a Matrix channel for it?
Maybe I don't know something about Matrix but I thought it had to be set up.

**Comment by Mic92** [comment]
We could just add a wiki category in the nixos discourse: https://discourse.nixos.org/
I don't think there that many discussions around the wiki that would justify a separate wiki.
For matrix there are irc bridges.

**Comment by Narice** [comment]
Alright I get what you where saying then 😉
Making a sub-section on the nixos discourse would be nice as it would make use of the same platform for everything! Maybe in the SIG section we could request a tag for wiki maintenance?
Or a documentation section which would also help the Nix/OS Project in general.
I'll try to make a thread and cross ping.

**Comment by Narice** [comment]
Oh! Rectification, there is already a documentation tag under the development category, I'll ask if it us possible to use that for the wiki instead
